tests/test_write_anything.py

```diff
@@ -36,7 +36,6 @@
     "SPDXYAMLExample-2.3.spdx.yaml-tag",
     "SPDXJSONExample-v2.3.spdx.json-tag",
     "SPDXXMLExample-v2.3.spdx.xml-tag",
-    "SPDXJsonExampleEmptyArrays.json-tag",
```

Comment:
Yey, seems like a conversion from json to tag with a very simple file is stable. :partying_face: